modules/content_filter/distance.py

```
@@ -0,0 +1,22 @@
+import math
+
+
+def cosine_similarity(a: list[float], b: list[float]) -> float:
+    """두 벡터의 코사인 유사도. 빈 벡터나 영벡터는 0.0."""
+    if not a or not b:
+        return 0.0
+    dot = sum(x * y for x, y in zip(a, b))
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)